R/data-gather-clean.R

```diff
@@ -0,0 +1,142 @@
+#' Get the submissions based on status
+#'
+#' Get the submissions based on status.
+#'
+#' @inheritParams mod_review_section_server
+#' @param statuses A character vector of statuses to
+#'   include from the set: "Accepted", "Rejected",
+#'   "In Review".
+get_submissions <- function(syn, group, statuses,
```

Comment:
Could you add documentation of the `group` argument?

Reply:
Can do! Thank you for checking these, by the way. It's so easy to miss the documentation errors.